src/request/components/request-view.scss

```diff
@@ -340,6 +340,42 @@ body, html {
     white-space: nowrap;
 }
 
+.request-logs-title {
+    font-size: 140%;
+    font-weight: normal;
+}
+
+.request-logs-title-count {
+    font-size: 140%;
+    font-weight: normal;
+    color: #616A6B; // grey
+}
+
+.request-logs-header {
+    font-weight: bold;
+    font-size: 100%;
+    border-top-style: none;
+    border-right-style: none;
+    border-left-style: none;
+    border-bottom-style: solid;
+    border-bottom-color: #AAB7B8;
+    border-bottom-width: 2px;
+}
```

Comment:
The above classes can probably be made a little more generic as we will probably end up using them outside of the log context. Hence `request-logs-title-count` = `request-title-count` and `request-logs-header` = `request-table-header`. 

Then anything that is log tab specific should be prefixed with `tab-log`, so the below classes will probably be updated to `tab-log-data-default`, `tab-log-data-warning` and `tab-log-data-error`.